src/js/table.js

```diff
@@ -269,7 +282,7 @@ function reallyDisplayOrders(orders, beautiful) {
                     // Remove the formatting to get integer data for summation
                     const floatVal = function(i) {
                         if(typeof i === 'string') {
-                            return (i === 'N/A' || i === '?') ?
+                            return (i === 'N/A' || i === 'n/a' || i === '?' || i === '-') ?
```

Comment:
we should wrap this in a function, perhaps called stringIsNullLike and stick it in util or extraction